packages/dynamic_layouts/lib/src/dynamic_grid.dart

```diff
@@ -27,13 +25,76 @@ class DynamicGridView extends GridView {
   /// Creates a scrollable, 2D array of widgets that are created on demand.
   DynamicGridView.builder({
     super.key,
+    super.scrollDirection,
+    super.reverse,
     required super.gridDelegate,
     // This creates a SliverChildBuilderDelegate in the super class.
     required IndexedWidgetBuilder itemBuilder,
     super.itemCount,
   }) : super.builder(itemBuilder: itemBuilder);
 
-  // TODO(snat-s): DynamicGridView.wrap?
+  /// Creates a scrollable, 2D array of widgets with tiles where each tile can
+  /// have its own size.
+  ///
+  /// Uses a [SliverGridDelegateWithWrapping] as the [gridDelegate].
+  ///
+  /// The following example shows how to use the DynamicGridView.wrap constructor.
+  ///
+  /// ```dart
+  /// DynamicGridView.wrap(
+  ///     mainAxisSpacing: 10,
+  ///     crossAxisSpacing: 20,
+  ///     children: [
+  ///         Container(
+  ///           height: 100,
+  ///           width: 200,
+  ///           color: Colors.amberAccent[100],
+  ///           child: const Center(child: Text('Item 1')
+  ///          ),
+  ///       ),
+  ///       Container(
+  ///           height: 50,
+  ///           width: 70,
+  ///           color: Colors.blue[100],
+  ///           child: const Center(child: Text('Item 2'),
+  ///          ),
+  ///       ),
+  ///       Container(
+  ///           height: 82,
+  ///           width: 300,
+  ///           color: Colors.pink[100],
+  ///           child: const Center(child: Text('Item 3'),
+  ///         ),
+  ///       ),
+  ///       Container(
+  ///           color: Colors.green[100],
+  ///           child: const Center(child: Text('Item 3'),
+  ///       ),
+  ///     ),
+  ///   ],
+  /// ),
+  /// ```
+  ///
+  /// See also:
+  ///
+  ///  * [SliverGridDelegateWithWrapping] to see a more detailed explanation of
+  ///    how the wrapping works.
+  DynamicGridView.wrap({
+    super.key,
+    super.scrollDirection,
+    super.reverse,
+    super.children = const <Widget>[],
+    double mainAxisSpacing = 0.0,
+    double crossAxisSpacing = 0.0,
+    double childCrossAxisExtent = double.infinity,
+    double childMainAxisExtent = double.infinity,
+  }) : super(
+            gridDelegate: SliverGridDelegateWithWrapping(
+          mainAxisSpacing: mainAxisSpacing,
+          crossAxisSpacing: crossAxisSpacing,
+          childCrossAxisExtent: childCrossAxisExtent,
+          childMainAxisExtent: childMainAxisExtent,
+        ));
```

Comment:
formatting here looks strange (especially line 92). I wonder, if it can be improved by placing a comma between the two `)` on this line.

Reply:
+1
```suggestion
        ),);
```